server/service.go

```diff
@@ -162,6 +163,14 @@ func CAKeyPassword(pw []byte) ServiceOption {
 	}
 }
 
+// optional argument to set the validity of signed client certs in days
+func ClientValidity(duration int) ServiceOption {
+	return func(s *service) error {
+		s.clientValidity = duration
```

Comment:
This is an optional argument, but it defaults to `0` if not set. That does not seem like a good default.

Please change the opts inside `NewService` so that a default time other than `0` is used. I'll leave it up to you to decide what a good option should be.